components/server/ee/src/gitlab/gitlab-app-support.ts

```diff
@@ -22,6 +22,13 @@ export class GitLabAppSupport {
         const api = new Gitlab({ oauthToken });
 
         const result: ProviderRepository[] = [];
+        const ownersRepos: ProviderRepository[] = [];
+
+        const identity = params.user.identities.find(i => i.authProviderId === "Public-GitLab");
```

Comment:
**issue**: This is related https://github.com/gitpod-io/gitpod/issues/5331, right? I'm still seeing a different top-level group selected by default when trying to add a new project from GitLab.com. Expected? 😭

Reply:
Oh no, that commit shouldn't go to that branch.

Reply:
removed